client/src/components/qa/QAModule.jsx

```diff
@@ -4,7 +4,6 @@ import SearchBar from './SearchBar';
 import QuestionList from './questionList/QuestionList';
 import AddQuestion from './addQuestion/AddQuestion';
 
-// eslint-disable-next-line react/prop-types
 export default function QAModule({ product }) {
   const [questionList, setQuestionList] = useState([]);
   const [sortList, setSort] = useState([]);
```

Comment:
Alright, I just skimmed through your PR and it looks good to me tbh lol, but TOM (hello if you're reading this) wants to see comments so I'm going to try to nitpick a bit. sozz